db/migrate/20150331143401_add_admin_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddAdminToUsers < ActiveRecord::Migration
+  def change
+    add_column :users, :admin, :boolean, default: false
```

Comment:
it is likely that you will need to define more than just an admin role. In general, I would suggest defining a `role` string column that is restricted to certain values. Your migration might look like this:

```ruby
add_column :users, :role, :string, default: 'user'
```